tests/unit/preprocessor/_multimodel/test_multimodel.py

```diff
@@ -540,6 +540,26 @@ def test_return_products():
     assert result4 == result2
 
 
+def test_ignore_tas_scalar_height_coord():
+    """Ignore conflicting aux_coords for height in tas."""
+    tas_2m = generate_cube_from_dates("monthly")
+    tas_1p5m = generate_cube_from_dates("monthly")
+
+    for cube, height in zip([tas_2m, tas_1p5m], [2., 1.5]):
+        cube.rename("air_temperature")
+        cube.attributes["short_name"] = "tas"
+        cube.add_aux_coord(
+            iris.coords.AuxCoord([height], var_name="height", units="m"))
+
+    result = mm.multi_model_statistics([tas_2m, tas_1p5m],
+                                       statistics=['mean'],
+                                       span='full')
+
+    # iris automatically averages the value of the scalar coordinate.
+    assert len(result['mean'].coords("height")) == 1
+    assert result["mean"].coord("height").points == 1.75
```

Comment:
I wonder what happens with, say, 2 cubes of 2m and one of 1.5? Is this somehow weighted? Might it make sense to do the test with three cubes (you can use `tas_2m.copy()` for one)?

Reply:
Interesting question. It turns out it will still return 1.75, i.e. the midpoint between the bounds that are set by the input cubes. I updated the top post accordingly.